assets/src/edit-story/components/statusCheck/statusCheck.js

```diff
@@ -0,0 +1,45 @@
+/*
+ * Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * External dependencies
+ */
+import { useCallback, useEffect } from 'react';
+/**
+ * Internal dependencies
+ */
+import { useAPI } from '../../app';
+
+function StatusCheck() {
+  const {
+    actions: { getStatusCheck },
+  } = useAPI();
+
+  const doStatusCheck = useCallback(() => {
```

Comment:
As mentioned in https://github.com/google/web-stories-wp/issues/4790#issue-713613396, to catch things like WAF blocking requests we'd need to send actual HTML markup here. Could be as simple as calling `getStoryMarkup()` on a stub story and sending that markup in the POST request.

And since `web-stories/v1/status-check` supports all request methods, we can probably even send two requests, one GET and one POST.

Reply:
I think we can add this later. It is the basics on this feature, we can improve later.